$NetBSD: patch-apps_CMakeLists.txt,v 1.1 2013/09/08 22:31:24 makoto Exp $

(1) WARN: PLIST:105: Man pages should be installed into man/, not share/man/.
(2) Avoid ${WRKSRC} included in RPATH.

--- apps/CMakeLists.txt.orig	2013-09-09 07:05:01.000000000 +0900
+++ apps/CMakeLists.txt	2013-09-09 07:00:40.000000000 +0900
@@ -94,7 +94,7 @@ endif()
 
 if(NOT WIN32) #rpath setup
 SET(CMAKE_SKIP_BUILD_RPATH  FALSE)
-SET(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE) 
+SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE) 
 SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
 endif()
@@ -121,5 +121,5 @@ if(NOT APPLE AND NOT WIN32)
 add_definitions("-DCONFDIR=\"${CMAKE_INSTALL_PREFIX}/share/TrustedQSL/\"")
 install(TARGETS tqsl DESTINATION bin)
 install(DIRECTORY help DESTINATION share/TrustedQSL)
-install(FILES tqsl.5 DESTINATION share/man/man5)
+install(FILES tqsl.5 DESTINATION man/man5)
 endif()
